cypress/support/fragments/data_import/dataImport.js

```diff
@@ -78,10 +78,83 @@ const importFile = (profileName, uniqueFileName) => {
   });
 };
 
+function uploadDefinitions(keyValue, fileName) {
+  return cy.okapiRequest({
+    path: 'data-import/uploadDefinitions',
+    body: { fileDefinitions: [{
+      uiKey: keyValue,
+      size: 2,
+      name: fileName
+    }] },
+    method: 'POST',
+    isDefaultSearchParamsRequired: false
+  });
+}
+
+function convertToBinery(fileName, uploadDefinitionId, fileId) {
```

Comment:
This in not only convert here
After converting file will be uploaded
I suggest to change function name for example uploadBinaryMarcFile or something else